lib/json_test.go

```diff
@@ -86,3 +88,50 @@ func TestOutputMetric(t *testing.T) {
 	assert.EqualValues(t, "", path)
 	assert.EqualValues(t, 0, value)
 }
+
+func TestCalcDiff(t *testing.T) {
+	dir, err := ioutil.TempDir("", "makerel-plugin-json-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+	os.Setenv("MACKEREL_PLUGIN_WORKDIR", dir) // to separate tempfile for each test
+
```

Comment:
Could you restore original "MACKERE_PLUGIN_WORKDIR" value with `defer`? It may cause adverse effects on other tests.

Reply:
Fixed at 3e5393b. Please confirm it.